falcon/response.py

```diff
@@ -132,6 +132,8 @@ class Response(object):
                 the Response instance itself (self).
 
         options (dict): Set of global options passed from the API handler.
+
+        headers (list): Read-only list of all headers set for Response.
```

Comment:
Let us update the docstrings to match the changes in design: _headers (~~list~~ dict):..._ etc